### PR TITLE
docs: ✏️ add nuxtjs example with TS and property-decorator

### PR DESCRIPTION
**Pull request by ozum**
Add nuxtjs example using class syntax with TypeScript and
nuxt-propert-decorator package

✅ Closes: #783

### CODE REVIEW

File: examples/nuxtjs-ts-property-decorator/package.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{
+  "name": "vue-styleguidist-example-nuxtjs",
```
Comment (elevatebart) [inline examples/nuxtjs-ts-property-decorator/package.json]:
```suggestion
  "name": "vue-styleguidist-example-nuxtjs-ts-property-decorator",
```

File: examples/nuxtjs-ts-property-decorator/package.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  "version": "4.0.0",
+  "description": "Use beside Nuxt.js and generate your styleguide besides your main app",
+  "keywords": [
+    "nuxt",
+    "nuxtjs",
+    "production ready",
+    "scaffolding"
+  ],
+  "author": "Knovour <knovour@gmail.com>",
```
Comment (elevatebart) [inline examples/nuxtjs-ts-property-decorator/package.json]:
```suggestion
  "author": "Ozum <ozum@gmail.com>",
```

**Comment by ozum** [comment]
> And can you add a comment about why we had to add the `empty.ts` file?

Oops. It is created by mistake. I deleted it and committed again with correct package.json data.

**Comment by elevatebart** [comment]
It actually was not, in order for a typescript project to work, you have to have at least one ts file.

**Pull request merged**